packages/react-router-dom-v5-compat/index.ts

```diff
@@ -69,7 +69,7 @@ export type {
   Pathname,
   Search,
   RoutesProps,
-} from "./react-router-dom";
+} from "../react-router-dom";
```

Comment:
We don't want to update these paths - they need to point to the local copy of `./react-router-dom` which is put there during the build - the reasoning is explained in the comment above these `exports`

Reply:
Actually - I see your note about the failing tests now.  That may just be because you hadn't run `yarn build` locally which copies these files into place.  I think you can just `git revert 71e8b7f` to undo the pathing changes here, then `yarn build && yarn test` and see if that fixes it 👍